kbase-extension/static/kbase/js/widgets/function_output/kbaseReportView.js

```diff
@@ -758,24 +612,13 @@ define([
                 // SUMMARY SECTION
```

Comment:
This section is really just a fancy one-liner, so it seemed redundant to decompose it into its own function, like the rest of the panels.